router/voteCallbacks.go

```diff
@@ -257,13 +305,136 @@ func (r *Router) getEnvelopeList(request routerRequest) {
 	if request.ListSize == 0 {
 		request.ListSize = MaxListSize
 	}
-	nullifiers := r.vocapp.State.EnvelopeList(request.ProcessID, request.From, request.ListSize, true)
+	envelopes, err := r.Scrutinizer.GetEnvelopes(request.ProcessID)
+	if err != nil {
+		r.sendError(request, fmt.Sprintf("cannot get envelope list: (%s)", err))
+		return
+	}
+	listBytes, err := proto.Marshal(envelopes)
+	if err != nil {
+		r.sendError(request, fmt.Sprintf("cannot marshal vote envelope list: (%s)", err))
+		return
+	}
 	var response types.MetaResponse
-	strnull := []string{}
-	for _, n := range nullifiers {
-		strnull = append(strnull, fmt.Sprintf("%x", n))
+	response.Content = listBytes
+	request.Send(r.buildReply(request, &response))
+}
+
+func (r *Router) getValidatorList(request routerRequest) {
+	var response types.MetaResponse
+	validators, err := r.vocapp.State.Validators(true)
+	if err != nil {
+		r.sendError(request, fmt.Sprintf("cannot get validator list: %v", err))
+		return
 	}
-	response.Nullifiers = &strnull
+	validatorList := new(models.ValidatorList)
+	validatorList.Validators = validators
+	validatorBytes, err := proto.Marshal(validatorList)
+	if err != nil {
+		r.sendError(request, fmt.Sprintf("cannot get validator list: %v", err))
+		return
+	}
+	response.ValidatorList = validatorBytes
```

Comment:
Can you explain what's this list please?

Reply:
ValidatorList is a protobuf model for a list of validators. I implemented some other protobuf wrappers/list models for this api as well, such as EnvelopePackage, EnvelopePackageList, VochainStats, BlockHeader, BlockHeaderList, TxPackage, TxPackageList. These models are just to serialize content for the api. I realize it doesn't make sense to have `response.ValidatorList` if we're doing this, I'll change it to use `response.Content`

Reply:
Let's use JSON for this list too. It's just 2 []byte fields, 1 int and 1 string. Its not worth it using protobuf here.